addons/account/models/account_bank_statement.py

```diff
@@ -188,7 +188,14 @@ def _default_journal(self):
     @api.depends('balance_start', 'previous_statement_id')
     def _compute_is_valid_balance_start(self):
         for bnk in self:
-            bnk.is_valid_balance_start = float_is_zero(bnk.balance_start - bnk.previous_statement_id.balance_end_real, precision_digits=bnk.currency_id.decimal_places)
+            bnk.is_valid_balance_start = (
+                float_is_zero(
+                    bnk.balance_start - bnk.previous_statement_id.balance_end_real,
+                    precision_digits=bnk.currency_id.decimal_places
+                )
```

Comment:
shorter: `bnk.currency_id.is_zero(bnk.balance_start - bnk.previous_statement_id.balance_end_real)`

Reply:
Edit: according to your commig message, it should be
`if bnk.previous_statement_id else True`

Reply:
thanks :+1: